recipes/hadolint/bld.bat

```diff
@@ -0,0 +1,3 @@
+set "BINARY_HOME=%PREFIX%\bin"
+mkdir "%BINARY_HOME%" || goto :error
```

Comment:
This requires a error section to be added, see e.g. https://github.com/conda-forge/shellcheck-feedstock/blob/master/recipe/bld.bat

Reply:
Thanks for the review! I have added the error section.